devtools_m1/cypress/integration/AdminCustomerInteractions.feature

```diff
@@ -0,0 +1,21 @@
+Feature: Admin Customer Interactions
```

Comment:
Do we ever test the unhappy scenerios? e.g.
Scenerio: An admin creates an account and the Drip API 500s

Reply:
We certainly should. I've been focused on smoke testing as a first step. I think that it'd be good to start making those tests as well.